src/main/java/com/notably/logic/suggestion/SuggestionCommand.java

```diff
@@ -0,0 +1,15 @@
+package com.notably.logic.suggestion;
+
+import com.notably.model.Model;
+
+/**
+ * Represents the Suggestion Command instance.
```

Comment:
Nit: Let's come up with a more descriptive description for the class?
For example, we can write: "Represents a particular strategy in generating suggestions"